defold-spine/editor/src/spineguiext.clj

```diff
@@ -159,8 +160,14 @@
             (value (gui/layout-property-getter clipping-inverted))
             (set (gui/layout-property-setter clipping-inverted)))
 
+  (property spine-create-bones g/Bool (protobuf/default Gui$NodeDesc :spine-create-bones)
```

Comment:
Hmm, I don't understand how it works. You didn't change gui_ddf.proto in the Defold repo; where is the `spine-create-bones` protobuf field coming from?

Reply:
Found https://github.com/defold/defold/pull/11307